gui/preload-chat.js

```diff
@@ -1060,7 +1067,7 @@ update_chat = function (obj) {
   }
 
   // check if wrong status message
-  if (obj.status == "transmit" && obj.type == "transmit" && obj.percent < 100) {
+  if (obj.status == "transmitting" && obj.type == "transmit" && obj.percent < 100) {
```

Comment:
I also had this for messages, which are at 100%. But we can test it with this approach at first.

Reply:
Resetting messages stuck in transmitting and equal to 100 should be reset after 6 hours when refreshing or relaunching already.  Should be close to this code block.